partner_email_check/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Email Format Checker',
+    'version': '11.0.1.0.0',
+    'summary': 'Validate email address field',
+    'author': "Komit, Odoo Community Association (OCA)",
+    'website': 'http://komit-consulting.com',
```

Comment:
Please change URL as described in https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L9

Reply:
Updated link: https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10